OpenUtau.Core/Format/USTx.cs

```diff
@@ -35,7 +36,15 @@ public class Ustx {
         public const string TENC = "tenc";
         public const string VOIC = "voic";
 
-        public static readonly string[] required = { DYN, PITD, CLR, ENG, VEL, VOL, ATK, DEC };
+        public static string[] GetRequiredExpressions(USinger? singer) {
+            if (singer == null) {
```

Comment:
Since in `ViewModels/ExpressionsViewModel.cs:61` you have
```cs
this.singer = singer ?? USinger.CreateMissing("");
```
to create "missing" singer instead of leaving the `singer` as `null`, and the `CreateMissing()` function is defined as following:
```cs
public static USinger CreateMissing(string name) {
    return new USinger() {
        found = false,
        loaded = false,
        name = name,
    };
}
```
you should check the `singer` value for its `Found` (or `Loaded`, I don't exactly know the difference) property instead of checking if it's `null`, like so:
```suggestion
        public static string[] GetRequiredExpressions(USinger singer) {
            if (singer.Found == false) {
```
This change should solve the "crash when opening the expressions menu with no singer" issue.